utility/src/main/java/org/oppia/util/parser/ImageLoader.java

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.target.SimpleTarget;
+
+public interface ImageLoader {
+
+  static void load(Context context, String path, int placeholder, SimpleTarget<Bitmap> target) {
```

Comment:
Doesn't this need to be an instance method? I was expecting that we need to mock out ImageLoader in tests to avoid using Glide in the actual test environment. That means we'd have something like:
- ``interface ImageLoader``
- ``class GlideImageLoader implements ImageLoader``
- Tests bind a Mockito mock of ``ImageLoader`` that performs the load operation
- Places that need ``ImageLoader`` inject it from Dagger
- We need a module to bind ``GlideImageLoader`` to ``ImageLoader`` so that injecting ``ImageLoader`` in production code corresponds to the glide image loader

Note that we don't need to pass ``Context`` in since subclasses of ``ImageLoader`` will be injectable and can inject the application context.

Reply:
@veena14cs This comment has not been addressed.

Reply:
I have made changes to the implementation part regarding ImageLoader. Testcases are still pending.

Reply:
Filed an issue #277 for this case.